Libraries/LibVT/Terminal.h

```diff
@@ -93,7 +93,24 @@ class Terminal {
         return m_lines[index];
     }
 
-    size_t max_history_size() const { return 500; }
+    size_t max_history_size() const { return m_max_history_lines; }
+    void set_max_history_size(size_t value)
+    {
+        if (m_max_history_lines > value) {
+            NonnullOwnPtrVector<Line> new_history;
+            new_history.ensure_capacity(value);
+            auto existing_line_count = min(m_history.size(), value);
+            for (size_t i = m_history.size() - existing_line_count; i < m_history.size(); ++i) {
+                auto j = (m_history_start + i) % m_history.size();
+                dbg() << "Copying index " << j << " into index " << i;
```

Comment:
If you want to keep this debug logging, at least please use `dbgln()` in new code :)

Reply:
Ah, sorry, don't need that